bindings/pyroot_experimental/PyROOT/python/ROOT/pythonization/_tcomplex.py

```diff
@@ -0,0 +1,46 @@
+# Author: Massimiliano Galli CERN  05/2019
+# Original PyROOT code by Wim Lavrijsen, LBL
+
+################################################################################
+# Copyright (C) 1995-2018, Rene Brun and Fons Rademakers.                      #
+# All rights reserved.                                                         #
+#                                                                              #
+# For the licensing terms see $ROOTSYS/LICENSE.                                #
+# For the list of contributors see $ROOTSYS/README/CREDITS.                    #
+################################################################################
+
+from ROOT import pythonization
+import cppyy
+
+def _rsub(self, other):
+    return -self+other
+
+def _rtruediv(self, other):
+    tcomplex = self.__class__
+    float_to_complex = tcomplex.TComplex(other,0)
```

Comment:
How do you know `other` is a `float`?

Reply:
And even if you check the types, does it make sense to implement `_rtruediv` for `TComplex`? What is the result of e.g. `2 / TComplex(1,0)`?